en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4217 @@
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#errorvalue">ErrorValue</a>]!</td>
+<td>
+
+A list containing unsuccessful usage values. Returns an empty array if no data is available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The granularity value for which data is retrieved.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorValue
+
+Represents the unsuccessful request count (i.e., 4xx and 5xx response codes) for a single timestamp. The timestamp is
+calculated based on the granularity value provided.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Represents a timestamp value that is calculated based on the specified granularity value. For example, granularity
+value is <code>1d</code>, the timestamp represents the start of the day calculated adhering to the timezone provided in the
+time filter.
+e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>errorCount</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Represents the error request count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorsByCategory
+
+Represents the errors by category over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>errors</strong></td>
+<td valign="top">[<a href="#errorcountbycategory">ErrorCountByCategory</a>]</td>
+<td>
+
+A list containing the error count for each error category over time. Returns an empty array if no data is available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The granularity value for which data is retrieved.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorsByStatusCode
+
+Represents the error counts grouped by status code for each API.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>errors</strong></td>
+<td valign="top">[<a href="#errorcountbycodeforapi">ErrorCountByCodeForAPI</a>]!</td>
+<td>
+
+A list containing the error count for each status code of each API. Returns an empty array if no data is available.
```

Comment:
Let's replace "of" with "for". Earlier, thought of avoiding the repetition of "for", but "status code of each API" seems to give a wrong meaning.